### PR TITLE
feature/UX-297 TimeDuration component

### PR DESCRIPTION
**Pull request by cliffmeyers**
Summary of changes:
* New component for expressing time durations, e.g. "2 minutes."
* Has a title "tooltip" that displays more precise duration such as "2 hrs, 52 mins, 45 secs"

### CODE REVIEW

File: src/js/components/TimeDuration.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * Also displays tooltip with more precise duration in "mos, days, hours, mins, secs" format.
+ * Tooltip text can be overridden via "hint" property.
+ *
+ * Properties:
+ * "millis": number or string.
+ * "hint": string to use for tooltip.
+ */
+export class TimeDuration extends Component {
+    render() {
+        const millis = !isNaN(this.props.millis) ?
```
Comment (sophistifunk) [inline src/js/components/TimeDuration.jsx]: This seems overly complicated. `parseInt(NaN)` returns NaN.

**Comment by sophistifunk** [comment]
One small comment inline, otherwise 🐝

**Comment by michaelneale** [comment]
BTW future PRs should get the build validated (was missing the account to give it permission to update the status).

**Comment by michaelneale** [comment]
looks good from eyeballing storybook :shipit:

File: src/js/components/TimeDuration.jsx (hunk 1 above)
Comment (cliffmeyers) [inline src/js/components/TimeDuration.jsx]: Yeah, trying to make this a bit too concise. Will clean this up tomorrow.

**Pull request merged**